torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -822,6 +839,12 @@ at::Tensor AtenXlaType::copy(const at::Tensor& src, bool /* non_blocking */,
   return bridge::CreateXlaTensor(CopyTensor(xla_tensors.front()), device);
 }
 
+at::Tensor& AtenXlaType::copy_(at::Tensor& self, const at::Tensor& src,
```

Comment:
This should be routed to TypeDefault, which then does the s_copy_ bounce.
If this was a simple copy, the TypeDefault would not be doing the s_copy_ dance.

Reply:
Done.